core/build.gradle

```diff
@@ -5,36 +5,38 @@ version = '0.15-SNAPSHOT'
 archivesBaseName = 'libdohj-core'
 
 dependencies {
-    compile 'org.bitcoinj:bitcoinj-core:0.15.9'
-    compile 'com.madgag.spongycastle:core:1.58.0.0'
+    implementation 'org.bouncycastle:bcprov-jdk15to18:1.68'
+    implementation 'org.bitcoinj:bitcoinj-core:0.15.10'
+    implementation 'com.google.guava:guava:30.0-android'
     implementation 'com.lambdaworks:scrypt:1.4.0'
-    implementation 'com.google.guava:guava:24.0-android'
-    compile 'com.google.protobuf:protobuf-java:3.5.1'
-    implementation 'com.squareup.okhttp3:okhttp:3.9.1'
-    implementation 'org.slf4j:slf4j-api:1.7.25'
+    implementation 'com.google.protobuf:protobuf-java:3.13.0'
+    implementation 'com.squareup.okhttp3:okhttp:3.12.12'
+    implementation 'org.slf4j:slf4j-api:1.7.30'
     implementation 'net.jcip:jcip-annotations:1.0'
-    compileOnly 'org.fusesource.leveldbjni:leveldbjni-all:1.8'
-    testImplementation 'junit:junit:4.12'
-    testImplementation 'org.easymock:easymock:3.2'
+    implementation 'org.fusesource.leveldbjni:leveldbjni-all:1.8'
```

Comment:
Not sure about that change. compileOnly is s till a thing even in newer Gradle versions.

Reply:
Reverted